src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,53 @@
+const key = 'DEV_KEY_HERE'; //<- change as needed
```

Comment:
I recommend the client side code to be in a separate PR that needs to be strictly merged after the back-end code is merged. Recommending this because, we will enforce any PR to also have unit tests and that will be a lot of line changes in one PR for reviewers to review.

Reply:
Should I be removing the client side code from this PR?

Reply:
Your choice. But it could potentially lead to a big PR with both java and js unit tests.

Reply:
Okay, I'm going to remove the comments.js file from this PR and reintroduce it in a different PR (or call comments from the backend instead).